pymeasure/instruments/pendulum/cnt91.py

```diff
@@ -119,12 +109,12 @@ def read_buffer(self, expected_length=0):
         map_values=True,
     )
 
-    measurement_time = Instrument.control(
+    gate_time = Instrument.control(
```

Comment:
If you change a property, please keep the old property and add a deprecation note. We do not want that someone's script breaks without notice.

For example:
~~~python
from warnings import warn

@property
def measurement_time(self):
    warn("measurement_time is deprecated, use `gate_time` instead.", FutureWarning)
    return self.gate_time
~~~

This should be noted in the changelog.md as well.

Reply:
I added deprecation warnings for setter/getter of `measurement_time` and for use of `sample_rate` in `buffer_frequency_time_series`. For the latter, I had to turn some arguments keyword-only by the use of `*`. This is might not be ideal for the use of either `gate_time` or `sample_rate` (users might want to specify this as a positional argument) but it guarantees that old scripts will accidentally use incorrect  gate times or sample rates (an exception is thrown in the case that `sample_rate` was provided as a positional argument and the user can change the script accordingly).

Reply:
Thanks.